packages/web-config-server/src/apiV1/measureBuilders/checkMultiConditionsByOrgUnit.js

```diff
@@ -0,0 +1,91 @@
+import { DataBuilder } from '/apiV1/dataBuilders/DataBuilder';
+import { checkValueSatisfiesCondition } from '@tupaia/utils';
+import { groupAnalyticsByOrgUnit } from './helpers';
+
+/**
+ * Example use case: If yes to SchCVD012 but no to SchCVD012a display 'Yes', if yes to SchCVD012 and yes to SchCVD012a display 'Yes, smart TV'
+ * Example orgUnitConditions config:
+ * orgUnitConditions: {
+    Yes: {
+      SchCVD012: {
+        value: 'Yes',
+        operator: '=',
+      },
+      SchCVD012a: {
+        value: 'No',
+        operator: '=',
+      },
+    },
+    'Yes, smart TV': {
+      SchCVD012: {
+        value: 'Yes',
+        operator: '=',
+      },
+      SchCVD012a: {
+        value: 'Yes',
+        operator: '=',
+      },
+    },
+    No: {
+      SchCVD012: {
+        value: 'No',
+        operator: '=',
+      },
+    },
+ */
+class CheckMultiConditionsByOrgUnit extends DataBuilder {
```

Comment:
I created this new measure builder to support a map overlay that has multi conditions to display values:

> Functioning TV, satellite receiver and dish set' to 'Functioning TV for teaching and learning purposes'
> (Responses: Yes
> Yes, smart TV
> No
> No data) (Codes: combines codes SchCVD012 and SchCVD012a) (If yes to SchCVD012 but no to SchCVD012a display Yes, if yes to SchCVD012 and yes to SchCVD012a display Yes, smart TV)